CONTRIBUTING.md

```diff
@@ -171,6 +171,18 @@ sbt
 applyCodeStyle
 ```
 
+To format Scala code more faster, you could format code with [Scala-CLI](https://scala-cli.virtuslab.org/) or [Coursier CLI](https://scalameta.org/scalafmt/docs/installation.html#cli):
+
+**With Scala-Cli**
+```shell
+scala-cli fmt
+```
+**With Coursier CLI**
+```Shell
+cs install scalafmt
+scalafmt --version # should be 3.7.17
```

Comment:
can you remove the version? - that will be a maintenance nightmare

Reply:
Ok.

Reply:
to match the scala-cli sample which actually formats - `scalafmt --version` doesn't format - it prints out the version

so could you just say `scalafmt` ?

Reply:
How about 
```
cs install scalafmt // skip it if scalafmt is already installed.
scalafmt --version 
```